src/Nethermind/Nethermind.Evm/EvmStack.cs

```diff
@@ -163,7 +163,8 @@ public void PushZero()
                 _tracer.ReportStackPush(ZeroStackItem());
             }
 
-            _bytes.Slice(Head * WordSize, WordSize).Clear();
+            // Clear the word
+            Unsafe.WriteUnaligned(ref Unsafe.Add(ref MemoryMarshal.GetReference(_bytes), Head * WordSize), Word.Zero);
```

Comment:
encapsulate as a method/extension method, can be aggressively inlined.

Reply:
Added as method and adjusted existing to use